### ISSUE TITLE
Remove google oauth dependency

### ISSUE THREAD

**Issue by fuse** [orientation/orientation#201]
It would be great to be agnostic about the oauth provider. For instance I'd like to be able to use orientation inside our company, connected to our own SSO.

I will take that in charge and propose a PR if you're interested in.

**Comment by olivierlacan** [comment]
Definitely. As long as we can work to make this configurable. Perhaps with custom Rails configuration options: http://edgeguides.rubyonrails.org/4_2_release_notes.html#railties-notable-changes
